plugins/main/public/components/add-modules-data/sample-data.tsx

```diff
@@ -51,7 +50,6 @@ const sampleSecurityInformationApplication = [
 
 const sampleThreatDetectionApplication = [
   vulnerabilityDetection.title,
-  virustotal.title,
```

Comment:
**nitpick**: the VirusTotal data should be visible through the Malware detection dashboard, so the sample data could be useful.